lib/large_object_store.rb

```diff
@@ -14,15 +14,16 @@ module LargeObjectStore
   RAW = 2
   FLAG_RADIX = 32 # we can store 32 different states
 
-  def self.wrap(store)
-    RailsWrapper.new(store)
+  def self.wrap(store, options = {})
+    RailsWrapper.new(store, options)
```

Comment:
I think this needs to be ** or it will fail on ruby 2.7/3
```suggestion
  def self.wrap(store, **options)
    RailsWrapper.new(store, **options)
```